### ISSUE TITLE
Add Flutter to existing apps，flutter engine leak or doesn't release on iOS platform

### ISSUE THREAD

**Issue by z234009184** [flutter/flutter#27149]
```
➜  ~ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.1.10-pre.193, on Mac OS X 10.14.2 18C54, locale zh-Hans-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.1)
[✓] Connected device (1 available)

• No issues found!
```

**Comment by z234009184** [comment]
Flutter engine doesn't release

**Comment by kangwang1988** [comment]
@z234009184 
Did you create the engine with: 
``` 
FlutterEngine *flutterEngine = [[FlutterEngine alloc] initWithName:@"io.flutter" project:nil];
```
If so, try
```
    FlutterEngine *flutterEngine = [[FlutterEngine alloc] initWithName:@"io.flutter" project:nil allowHeadlessExecution:NO];
```

**Comment by kangwang1988** [comment]
Closed #27149 via https://github.com/flutter/engine/pull/7610

**Comment by lihui2333** [comment]
when  should  i call the destoryContext method to  release the flutterviewcontroller?
